src/components/chart/state-history-chart-timeline.ts

```diff
@@ -32,18 +32,26 @@ export class StateHistoryChartTimeline extends LitElement {
 
   @property({ attribute: false }) public endTime!: Date;
 
+  @property({ type: Number }) public paddingYAxis = 0;
+
+  @property({ type: Number }) public index?;
```

Comment:
Can you use a more descriptive name for `index` here so better shows the intent?

Reply:
Hm, having trouble of thinking of anything too descriptive. This is just for the state-history-charts wrapper, when it instances N different charts, it assigns each one a number from 0..N-1, so that when they fire the widthChanged callback, it can keep track of which one sent the event so it can uniquely track each of their widths. 

Maybe `childIndex`? Not sure if that's any better. Or maybe I can just add a descriptive comment?

Reply:
Even `chartIndex` would be fine here so its clear its the index of charts and not an index of the data

Reply:
Changed as requested.